src/Compilers/Core/MSBuildTask/CopyRefAssembly.cs

```diff
@@ -66,6 +66,8 @@ public override bool Execute()
                             Log.LogMessageFromResources(MessageImportance.Low, "CopyRefAssembly_SkippingCopy1", DestinationPath);
                             return true;
                         }
+
+                        Log.LogMessageFromResources(MessageImportance.Low, "CopyRefAssembly_Changed", SourcePath, File.GetLastWriteTimeUtc(SourcePath).ToString("O"), source, DestinationPath, File.GetLastWriteTimeUtc(DestinationPath).ToString("O"), destination);
```

Comment:
>MessageImportance.Low [](http://example.com/codeflow?start=52&length=21)

Should the `MessageImportance` be a higher value?

Reply:
I went with Kirill's recommendation (see linked issue)

Reply:
Ah, this is a success case, not a failure case. That makes sense.